content/news/028/index.md

```diff
@@ -135,6 +135,46 @@ They also worked on a [tutorial][bitgun-tutorial].
 [bitgun-missions]: https://twitter.com/LogLogGames/status/1464009563976392713?s=20
 [bitgun-tutorial]: https://twitter.com/LogLogGames/status/1461898845810348033?s=20
 
+### [Veloren][veloren]
+
+![Travelling merchant](veloren.jpg)
+_A merchant paused in their travels_
+
+[Veloren][veloren] is an open world, open-source voxel RPG inspired by Dwarf
+Fortress and Cube World.
+
+In November, Veloren started a new initiative to help new developers learn more
+about the code base; the Veloren Reading Club. You can watch the
+[first][veloren-reading-club-1] and [second][veloren-reading-club-2] sessions
+now. Hats were merged, and  a major rewrite of the server-hosting section of the
```

Comment:
```suggestion
now. Hats were merged, and a major rewrite of the server-hosting section of the
```

Reply:
I told myself I would go back to this while I was racing through the content, but I suppose I forgot :smile: thanks for the catch!